cms/templates/cms/program_page.html

```diff
@@ -5,6 +5,8 @@
 {% load wagtailimages_tags %}
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
```

Comment:
Why not `{{ page.title }}`? or `{{ page.title }} MicroMasters` to be consistent with the actual content of the page.